app/src/main/res/values-de/strings.xml

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string name="format_minutes">%1$dm</string>
+    <string name="format_hours">%1$dh</string>
```

Comment:
If I remember correctly, hours are in German 'Stunden', so this string still needs translation I think.

Reply:
Okay, I did it.

Reply:
No, you shouldn’t change the name of the string, because that is just the name that the code uses to get the translation.

Reply:
UPS I have undone it